### PR TITLE
ReadMe Updated

### PR DESCRIPTION
**Pull request by Sonichigo**
Signed-off-by: sonichigo <kurosakiichigo.songoku@gmail.com>

### CODE REVIEW

File: echo-sql/README.md [hunk 1]
```diff
@@ -3,6 +3,12 @@
 A sample url shortener app to test Keploy integration capabilities using [Echo](https://echo.labstack.com/) and PostgreSQL. 
 
 ## Setup
+### Start MongoDB
+Spin up MongoDB to store the test-runs results
```
Comment (nehagup) [inline echo-sql/README.md:7]: Keploy doesn't need MongoDB anymore.

File: echo-sql/README.md [hunk 2]
```diff
@@ -35,8 +41,8 @@ git clone https://github.com/keploy/samples-go && cd samples-go/echo-sql
 # start Postgres
 docker-compose up -d
 
-# run the sample app in record mode
-export KEPLOY_MODE=record && go run handler.go main.go
```
Comment (nehagup) [inline echo-sql/README.md:39]: why is exporting of env var removed?

**Review by nehagup — CHANGES_REQUESTED** [review]

File: echo-sql/README.md (hunk 2 above)
Comment (Sonichigo) [reply echo-sql/README.md:39]: My mistake it got mixed up with MongoDB section, and got removed. Fixed it

**Review by nehagup — APPROVED** [review]
LGTM! 👏

**Pull request merged**